pb/config/config.proto

```diff
@@ -328,7 +330,7 @@ message TestGroup {
   // These can be substituted into LinkTemplates.
   string user_property = 56;
 
-  reserved 58,59;
+  reserved 57,58,59;
```

Comment:
Does 57-59 work? Do we care? (maybe for next PR)

Reply:
Good question, I believe comma-separated or range can be mix-matched (https://developers.google.com/protocol-buffers/docs/proto3#reserved). Happy to do a range instead if that reads clearer.